src/controllers/itemDetails/index.js

```diff
@@ -1319,7 +1319,9 @@ function renderChildren(page, item) {
         Fields: fields
     };
 
-    if (item.Type !== 'BoxSet') {
+    if (item.Type == 'MusicAlbum') {
```

Comment:
It might be more logical to have this on line no. 1344 where we do something similar for `MusicArtist`.

Reply:
Maybe all those cases could be merged, yes. I just wanted to keep the two `SortBy` cases together.